testcontainers/cassandra.py

```diff
@@ -0,0 +1,46 @@
+from testcontainers.core.config import MAX_TRIES
+from testcontainers.core.generic import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready, wait_for_logs
+
+
+class CassandraContainer(DockerContainer):
+    """
+    Cassandra database container.
+
+    Example
+    -------
+    ::
```

Comment:
Sorry, one more. Can we add `.. doctest::` here (and the other container) to make things consistent with #268.

Reply:
done.

we are 10 days from this PR birthday, can we have a celebration ?

Reply:
@tillahoffmann Can we please finalize this?

Reply:
@fruch While we are waiting for it, seems that both new databases should be added to `README.rst` as well.